File: android/src/main/kotlin/br/com/cpndntech/google_maps_playback/GoogleMapsPlaybackPlugin.kt
package br.com.cpndntech.google_maps_playback

import io.flutter.embedding.engine.plugins.FlutterPlugin

class GoogleMapsPlaybackPlugin : FlutterPlugin {
    override fun onAttachedToEngine(flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
        flutterPluginBinding.platformViewRegistry.registerViewFactory(
            "br.com.cpndntech.google_maps_playback/playback",
            GoogleMapsPlaybackFactory(flutterPluginBinding.binaryMessenger)
        )
    }

    override fun onDetachedFromEngine(binding: FlutterPlugin.FlutterPluginBinding) {}
}

File: ios/Classes/GoogleMapsPlaybackPlugin.swift
import Flutter
import UIKit

public class GoogleMapsPlaybackPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let factory = GoogleMapsPlaybackFactory(messenger: registrar.messenger())
    registrar.register(factory, withId: "br.com.cpndntech.google_maps_playback/playback")
  }
}
